src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -67,6 +68,8 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
   columns: CdTableColumn[];
   clusterWideActions: CdTableAction[];
   icons = Icons;
+  osdNearFullRatio: number;
+  osdFullRatio: number;
```

Comment:
Rather than defining in multiple places these 2 fields, why not specifying an `OSDSettings` class containing these 2 fields?

Reply:
Done